packages/monaco/src/browser/monaco-workspace.ts

```diff
@@ -364,4 +373,20 @@ export class MonacoWorkspace {
             }
         }
     }
+
+    private async performSnippetEdits(edits: MonacoResourceTextEdit[]): Promise<void> {
```

Comment:
Minor: To keep it as simple as possible to override services, Theia never uses `private`, but keeps everything `protected`.